tests/system/runtimes/test_databricks.py

```diff
@@ -118,6 +119,32 @@ def print_kwargs(**kwargs):
                 == "kwargs: {'param1': 'value1', 'param2': 'value2'}\n"
             )
 
+    def test_failure_in_databricks(self):
+        code = """
+
+def import_mlrun():
+    import mlrun
+"""
+
+        function_ref = FunctionReference(
+            kind="databricks",
+            code=code,
+            image="mlrun/mlrun",
+            name="databricks-fails-test",
+        )
+
+        function = function_ref.to_function()
+
+        self._add_databricks_env(function=function, is_cluster_id_required=True)
+        with pytest.raises(RunError) as error:
+            function.run(
+                handler="import_mlrun",
+                project="databricks-proj",
+            )
+        lines = str(error.value).splitlines()
+        assert "No module named 'mlrun'" in lines[2]
```

Comment:
I ran it and it failed on
```
>           assert "No module named 'mlrun'" in lines[2]
E           IndexError: list index out of range
```

Reply:
Have you upgraded your server to this mlrun PR version?

Reply:
Got it.